### DISCUSSION TITLE
Data Quality in Explore

### DISCUSSION THREAD

**Discussion by richardphi1618** [lightdash/lightdash#4362] [Q&A]
Is there a way to represent data quality in the explore views? I see they are working on freshness but dbt tests pass/fail statistics would be helpful too. 

Maybe I'm not asking the right question or this already exists?

**Comment by hamzahc1** [comment]
Hi @richardphi1618 - I think we could include something here for sure.  One simple way would be to include the dbt tests as part of the compile process when using the [Lightdash CLI tool](https://docs.lightdash.com/guides/cli/intro). That way you'd know before you update Lightdash what's working and what isn't. Would that potentially work? 

Also tagging @owlas to add some thoughts too!
